### DISCUSSION TITLE
remoteEntry.js modules not to be cached

### DISCUSSION THREAD

**Discussion by aquarazorda** [webpack/webpack#12986] [Q&A]
Hello, I'm trying to load remoteEntry.js once and then use it to load same or different modules in different places, but the issue that I'm facing is that, modules get cached. So for example if I load the module second time, it has the state of first load and I can't avoid that.

With current loadRemoteModule() behavior, if I load same module with different data twice on the same page, I get two remoteEntry.js added to the DOM, so I overloaded it and if I already have remoteEntry.js present, I do not add it again. With this approach, I'm facing that module cache problem.

What I'm trying to achieve is, that I want to load remoteEntry.js once and use it to load modules in different places as many times as I want.

Thanks

**Comment by XiaofengXie16** [comment]
You may be interested in this plugin to invalidate cache.
https://github.com/module-federation/module-federation-examples/issues/566

**Reply by aquarazorda** [reply]
Looks interesting, will check it out, hopefully it will help me solve this issue.

Thanks for the response!

**Reply by aquarazorda** [reply]
So if I load same remoteEntry.js with different additional parameters assigned to the url, I have to make that same request and load that js file multiple times, while I want to load js file only once and every time I initialize it, I want it to be initialized with clear state and not with the state of last load.

Plus I'm not loading it with webpack.config.js, I load modules dynamically with loadRemoteModule() function.
